src/components/ReportWelcomeText.tsx

```diff
@@ -42,7 +43,7 @@ function ReportWelcomeText({report, policy, personalDetails}: ReportWelcomeTextP
     const participantAccountIDs = ReportUtils.getParticipantsAccountIDsForDisplay(report);
     const isMultipleParticipant = participantAccountIDs.length > 1;
     const displayNamesWithTooltips = ReportUtils.getDisplayNamesWithTooltips(OptionsListUtils.getPersonalDetailsForAccountIDs(participantAccountIDs, personalDetails), isMultipleParticipant);
-    const roomWelcomeMessage = ReportUtils.getRoomWelcomeMessage(report);
+    const welcomeMessage = SidebarUtils.getWelcomeMessage(report, policy);
```

Comment:
We are still using `policy.description` and `report.description`. Please use `welcomeMessage.messageHtml` instead

Reply:
I updated.